docs/articles/migration/4x_to_5x.md

```diff
@@ -0,0 +1,91 @@
+---
+uid: articles.migration.4x_to_5x
+title: Migration 4.x - 5.x
+---
+
+## Migration from 4.x to 5.x
+
+> ![NOTE]
+> The API surface of DSharpPlus 5.x is not stable yet. This migration guide may be incomplete or outdated. It is recommended to cross-reference with the [tracking issue](https://github.com/DSharpPlus/DSharpPlus/issues/1585) when migrating.
+
+The first change you will likely encounter is a rewrite to how bots are set up. We now support two approaches to the problem instead of the old approach:
+
+# [DiscordClientBuilder](#tab/discordclientbuilder)
+
+The simplest way to get a bot running is to use `DSharpPlus.DiscordClientBuilder`. To get started, create a new builder as follows: 
+
+```cs
+DiscordClientBuilder builder = DiscordClientBuilder.Default(string token, DiscordIntents intents);
```

Comment:
Perhaps the method should be called `CreateDefault`?

Reply:
we could, i don't have any strong opinions on it